### DISCUSSION TITLE
Enabling repeat does not work

### DISCUSSION THREAD

**Discussion by AMBARCAPOOR** [gopro/labs#1036] [Q&A]
I am trying to program the gopro 9 to be activated when it senses motion Ave do a burst photo. When I select repeat, it only does the function one time. I would love to have this repeat every time it senses action between 6am and 7pm.
![Screenshot_20240823_120637](https://github.com/user-attachments/assets/7bcf2c60-26c2-4d52-b23d-d4a9c90834d8)

**Comment by dnewman-gpsw** [comment]
Motion detection is designed for video, I don't know if you will be able to get to work with Burst reliably.

Also you are using repeat in less than optimal way.  Separate QR Code for your settings than your actions, otherwise your repeat will re-setup you settings on every loop.

**Comment by AMBARCAPOOR** [comment]
Thank you. Let's say I go the video route instead. How would I set it up
for video to record when the hummingbird is in the frame and stop when it
leaves.

I didn't understand the second part of your message, I think autocorrect
got involved, lol. Could you please explain what you meant by separate qr
codes for settings and action?

Thank you!

**Reply by dnewman-gpsw** [reply]
Use setting in one QR code, thing like from Mode, Protune, and Prefs, but any action, like capture, use a separate QR Code.

More on motion detection https://gopro.github.io/labs/control/motion/

Just experiment.

**Comment by AMBARCAPOOR** [comment]
Gotcha. That makes a lot of sense! Thank you boss. 🙏🏾
